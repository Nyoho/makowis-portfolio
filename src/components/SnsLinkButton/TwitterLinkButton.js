// @flow
import React from 'react';
import BaseButton from './BaseButton';

const TwitterLinkButton = (props: { url: string }) => {
  const btnstyle = {
    background: '#1da1f3',
  };
  return <BaseButton url={props.url} icon="twitter" btnstyle={btnstyle} />;
};

export default TwitterLinkButton;
