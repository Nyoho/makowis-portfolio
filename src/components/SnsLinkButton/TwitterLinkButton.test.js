/* eslint-env jest */
// @flow
import React from 'react';
import renderer from 'react-test-renderer';
import TwitterLinkButton from './TwitterLinkButton';

it('renders TwitterLinkButton', () => {
  const tree = renderer
    .create(<TwitterLinkButton twitterId="mako_wis" />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
