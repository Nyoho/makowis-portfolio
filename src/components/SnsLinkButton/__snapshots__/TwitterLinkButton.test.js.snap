// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders TwitterLinkButton 1`] = `
<a
  className="SnsLinkButton"
  href="https://twitter.com/mako_wis"
  style={
    Object {
      "background": "#1da1f3",
    }
  }
/>
`;
